packages/central-server/src/apiV2/surveyResponse.js

```diff
@@ -146,7 +146,6 @@ export async function surveyResponse(req, res) {
 }
 
 const constructAnswerValidators = models => ({
-  id: [hasContent, takesIdForm],
```

Comment:
Could we keep the validation for `takesIdForm` if it is present? Mabye:
`id: [constructIsNotPresentOr(takesIdForm)]`